xen/xenops_interface.ml

```diff
@@ -433,6 +433,8 @@ module Vif = struct
 
 	type id = string * string
 
+	type static_ip_setting = (string * string) list
```

Comment:
I do not understand the meaning of "Static", could you please explain a bit more about it to me?

If I do "open Xenops_interface" inside xen-api, is "Static" visible inside xen-api code?